Examples/Python/GroomUtils.py

```diff
@@ -155,12 +155,16 @@ def FindReferenceImage(inDataList):
     print(" ")
     return inDataList[idx]
 
-def applyResampling(outDir, refFile, inDataListSeg, inDataListImg=[]):
+def applyRigidAlignment(outDir, refFile, inDataListSeg, inDataListImg=[]):
     """
     This function takes in a filelists(binary and raw) and makes the 
     size and spacing the same as the reference
     """
-    print("\n############# Resample #############")
+    isoValue       = 1e-20
+    icp_iterations = 200
```

Comment:
icp iterations should be added as input argument with a default? 100 vs 500 makes a difference in the alignment result based on how misaligned the shapes are.

Reply:
Added 2d12d22